ln.py

```diff
@@ -116,7 +116,9 @@ def list_invoices(self):
             description = invoice["description"]
             payment_hash = invoice["payment_hash"]
             expires_at = invoice["expires_at"]
-            amount_msat = invoice["amount_msat"]
+            amount_msat = 0
+            if amount_msat:
```

Comment:
this condition will never be true (```amount_msat = 0```)

Reply:
reverted back to line 119 when there is no amount_msat = 0 and condition and getting an error:
```
app_1                       |     received_txs = rpc.list_invoices()
app_1                       |   File "/app/ln.py", line 119, in list_invoices
app_1                       |     amount_msat = invoice["amount_msat"]
app_1                       | KeyError: 'amount_msat'
```

will look at it further and see if i can figure this out.

Reply:
since this is transaction related, changed it.